src/cloud/docker/docker-manage-cron-jobs.md

```diff
@@ -0,0 +1,49 @@
+---
```

Comment:
This page lacks content on the fact that crons can be managed through `.magento.app.yaml`.